scripts/check_teams.py

```diff
@@ -14,110 +25,214 @@
 
 HERE = Path(__file__).parent
 ROOT = HERE.parent
-yaml = YAML(typ="safe")
-
+yaml = YAML()
+yaml.indent(mapping=2, sequence=4, offset=2)
+
+
+def eprint(*args, **kwargs):
+    kwargs.setdefault("file", sys.stderr)
+    print(*args, **kwargs)
+
+
+def report_diff(field: str, **entries: list[str]):
```

Comment:
It would be nice if `report_diff` handles both collections and single items for us automatically. For example, a nice output would be:

For single values, notice it isn't wrapped in a list:
```
Contents for description in file do not match github:
file: Maintainers of conda-pypi
github: Maintainers of `conda-pypi`
```

For collections:
```
Contents for members in file do not match github:
file: ['alice', 'bob', 'charlie']
github: ['alice', 'charlie', 'dan']
```

Then the caller could more directly call the function like:

```
report_diff("descriptions",
    file=team["description"],         # Natural: passing a string
    github=details["description"]     # Let function decide how to format
)
```